pinot-core/src/main/java/org/apache/pinot/core/minion/SegmentPurger.java

```diff
@@ -85,7 +85,10 @@ public File purgeSegment()
       }
 
       Schema schema = purgeRecordReader.getSchema();
-      SegmentGeneratorConfig config = new SegmentGeneratorConfig(schema);
+      // FIXME: figure out how to get table config here.
```

Comment:
create an  issue for this and add the issue number here?

Reply:
Was able to fix it. please take a look

Reply:
As per offline discussion with @Jackie-Jiang  removing the timeColumnName from SegmentPurger for now. He will be working on a fix to make TableConfig available to SegmentPurger very soon.